com.reprezen.swagedit.openapi3/resources/templates.xml

```diff
@@ -0,0 +1,328 @@
+<templates>
+	<template name="OpenApi version" 
+       id="com.reprezen.swagedit.openapi3.templates.openapi_template" 
+       description="OpenAPI version declaration" 
+       context="com.reprezen.swagedit.openapi3.templates.root" 
+       enabled="true">openapi: "3.0"</template>
```

Comment:
Should be changed to "3.0.0" as the `openapi` property [uses semantic versioning](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#schema):
> Specifies the OpenAPI Specification version being used <...>The structure SHALL be major.minor.patch